### PR TITLE
Expose --v in oc

### PR DESCRIPTION
**Pull request by soltysh**
We can't expose `-v` yet, since some commands are already using that shorthand. I've deprecated them for now and we'll enable shorthand as soon as possible. 

/assign @deads2k @juanvallejo

### CODE REVIEW

File: pkg/oc/cli/cmd/process.go [hunk 1]
```diff
@@ -88,10 +88,7 @@ func NewCmdProcess(fullName string, f *clientcmd.Factory, in io.Reader, out, err
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to read a template")
 	cmd.MarkFlagFilename("filename", "yaml", "yml", "json")
-	params := cmd.Flags().StringArrayP("value", "v", nil, "Specify a key-value pair (eg. -v FOO=BAR) to set/override a parameter value in the template.")
```
Comment (soltysh) [inline pkg/oc/cli/cmd/process.go]: This was deprecated in 1.5 (over a year ago) so I think we're safe entirely dropping it.

Comment (deads2k) [reply pkg/oc/cli/cmd/process.go]: > This was deprecated in 1.5 (over a year ago) so I think we're safe entirely dropping it.

@liggitt you're more conservative than me.  Arguments against?

Comment (deads2k) [inline pkg/oc/cli/cmd/process.go]: Short flags conflict with long flags? I didn't expect that.

Comment (deads2k) [reply pkg/oc/cli/cmd/process.go]: > Short flags conflict with long flags? I didn't expect that.

huh.  I didn't realize that it registered `--v` and -v`.  That's suprising.

File: pkg/oc/cli/cmd/status.go [hunk 2]
```diff
@@ -86,7 +86,10 @@ func NewCmdStatus(name, baseCLIName, fullName string, f *clientcmd.Factory, out
 	}
 
 	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", opts.outputFormat, "Output format. One of: dot.")
-	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", opts.verbose, "See details for resolving issues.")
+	cmd.Flags().BoolVarP(&opts.suggest, "verbose", "v", opts.suggest, "See details for resolving issues.")
+	cmd.Flags().MarkDeprecated("verbose", "Use --suggest instead.")
+	cmd.Flags().MarkHidden("verbose")
+	cmd.Flags().BoolVarP(&opts.suggest, "suggest", "s", opts.suggest, "See details for resolving issues.")
```
Comment (deads2k) [inline pkg/oc/cli/cmd/status.go]: no shorthand.  They are scarce

File: pkg/oc/cli/cmd/process.go (hunk 1 above)
Comment (liggitt) [reply pkg/oc/cli/cmd/process.go]: drop it

File: pkg/oc/cli/cmd/status.go (hunk 2 above)
Comment (liggitt) [reply pkg/oc/cli/cmd/status.go]: why not just drop the shorthand and use "verbose"?

File: pkg/oc/cli/cmd/process.go (hunk 1 above)
Comment (soltysh) [reply pkg/oc/cli/cmd/process.go]: Yeah, there's a `AddGoFlags` call, which adds a shorthand `-v` for `--v`. So for now we can't have that shorthand. We only can use the slightly longer one, but slowly getting there ;)

Comment (soltysh) [reply pkg/oc/cli/cmd/process.go]: See my other comment.

File: pkg/oc/cli/cmd/status.go (hunk 2 above)
Comment (soltysh) [reply pkg/oc/cli/cmd/status.go]: This is not verbosity level in any way, this is a boolean flag whether to show suggestions. That's why this better reflects the intent.

**Comment by soltysh** [comment]
Dropped the short 's' David asked for and fixed the tests.

**Comment by deads2k** [comment]
Add `test-cmd` tests for a few `oc` commands to be sure that the new loglevel flags work.

**Comment by deads2k** [comment]
> Add test-cmd tests for a few oc commands to be sure that the new loglevel flags work.

then lgtm

/approve

**Comment by soltysh** [comment]
Tests are there, so self-lgtm based on previous approval.

**Comment by deads2k** [comment]
/cherrypick release-3.9

@soltysh Don't merge the pick.  We'll have to talk about it there.

**Comment by raffian** [comment]
Found this issue after realizing `oc --v=6 (or -v=7,8,9)` does not appear to enable http debugging in Openshift 3.6 for viewing request headers and content for the requested resource; is it turned off for a reason?

**Comment by soltysh** [comment]
@raffian we've only recently added `--v` support in `oc`. Up until 3.9 you need to use `--loglevel` instead.

**Comment by raffian** [comment]
@soltysh Thank you, are the level values consistent between `--v` and `--loglevel`?

**Comment by soltysh** [comment]
@raffian yes, they can be used interchangeably.

**Pull request merged**